src/layouts/SiteLaunchPad/components/SiteLaunchDisclaimer.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  Box,
+  Icon,
+  ListItem,
+  OrderedList,
+  Text,
+  UnorderedList,
+} from "@chakra-ui/react"
+import { Button } from "@opengovsg/design-system-react"
+import { BiRightArrowAlt } from "react-icons/bi"
+
+import { SITE_LAUNCH_PAGES } from "types/siteLaunch"
+
+import { SiteLaunchPadBody } from "./SiteLaunchPadBody"
+import { SiteLaunchPadTitle } from "./SiteLaunchPadTitle"
+
+export const SiteLaunchDisclaimerTitle = (): JSX.Element => {
+  const title = "Welcome to the launchpad"
+  const subTitle = "Track and manage your site launch here"
+  return <SiteLaunchPadTitle title={title} subTitle={subTitle} />
+}
+
+interface SiteLaunchDisclaimerBodyProps {
+  setPageNumber: (number: number) => void
+}
+
+export const SiteLaunchDisclaimerBody = ({
+  setPageNumber,
+}: SiteLaunchDisclaimerBodyProps): JSX.Element => {
+  return (
+    <SiteLaunchPadBody>
+      <Text as="h2" textStyle="h2">
```

Comment:
@kishore03109 might want to check if these heading tags need to be remapped

ref: https://github.com/isomerpages/isomercms-frontend/pull/1202

```
Visual Changes
Due to the upgrade of design system, the headings have been remapped.

Previous H6 are now h4
Previous H5 are now h3
Previous H4 are now h2
```

Reply:
ye thanks for the heads up, [remapped alr](https://github.com/isomerpages/isomercms-frontend/pull/1308/commits/5aa4647d8ee0822769087cf7263ac9cc1f343e82)